sass/_layout.scss

```diff
@@ -495,29 +514,36 @@ footer {
                 }
             }
         }
+
+        @include mobile {
+            top: auto;
+            @include transform(translateY(0));
+        }
+    }
+
+    .footer-copyright {
+        position: relative;
+        top: 50%;
+        @include transform(translateY(-50%));
+
+        @include mobile {
+            top: auto;
+            @include transform(translateY(0));
+        }
     }
 
     .footer-social {
         li {
-            margin: 0 $baseline;
+            margin: 0 .5vw;
```

Comment:
@middric is it alright to use this `vw` value or should I strictly use the variables for margins/padding?